.gitignore

```
@@ -18,6 +18,7 @@ cache/
 ./**/*.parquet
 
 ./data/*
+./**/data/*
```

I just randomly noticed this and wanted to mention two things:

First, it is not necessary to make separate entries for directories with the same name at different levels. One entry using the syntax "directory-name/" will ignore every directory with that name (plus everything underneath it) no matter where it is in the repo. So the two lines here can be replaced with one, like this:

```suggestion
data/
```

Second, the .gitignore file the main branch has already taken care of this. I suggest using it instead.

---

this PR is very much outdated, the rebased version is linked above in my branch. However, I suspect that maybe outdated already as well.

---

branch is here: https://github.com/fornax-navo/fornax-demo-notebooks/pull/172#issuecomment-1835503967